backends/platform/symbian/src/SymbianOS.cpp

```diff
@@ -92,6 +89,15 @@ void OSystem_SDL_Symbian::initBackend() {
 	if (!ConfMan.hasKey("savepath")) {
 		ConfMan.set("savepath", savePath);
 	}
+	
+#if !RELEASE_BUILD
+	_LIT(KDefaultBetaExtraPath,"!:\\DATA\\ScummVM\\BETA\\");
```

Comment:
Is it supposed to be in this hardcoded directory?

Reply:
Yes. First letter - '!' placeholder for real drive name. Drive name can be from A to Y. In that folder placed all datafiles for beta versions by installer.
Currently datafiles doesn't fit in phone memory.